src/TLuaInterpreter.cpp

```diff
@@ -5724,6 +5798,197 @@ int TLuaInterpreter::loadSoundFile(lua_State* L)
     return loadMediaFileAsOrderedArguments(L);
 }
 
+// Private
+int TLuaInterpreter::playVideoFileAsOrderedArguments(lua_State* L)
+{
+    Host& host = getHostFromLua(L);
+    TMediaData mediaData{};
+    int numArgs = lua_gettop(L);
+    QString stringValue;
+    int intValue = 0;
+    bool boolValue = 0;
+
+    // name[,volume][,start][,loops][,key][,tag][,continue][,url])
+    for (int i = 1; i <= numArgs; i++) {
+        if (lua_isnil(L, i)) {
+            continue;
+        }
+
+        switch (i) {
+        case 1:
+            stringValue = getVerifiedString(L, __func__, i, "name");
+
+            if (QDir::homePath().contains('\\')) {
+                stringValue.replace('/', R"(\)");
+            } else {
+                stringValue.replace('\\', "/");
+            }
+
+            mediaData.setMediaFileName(stringValue);
+            break;
+        case 2:
+            intValue = getVerifiedInt(L, __func__, i, "volume");
+
+            if (intValue == TMediaData::MediaVolumePreload) {
+                {
+                } // Volume of 0 supports preloading
```

Comment:
You already have a set of braces - no need for another pair:
```suggestion
                // Volume of 0 supports preloading
```